### ISSUE TITLE
[Bug]: Checkbox fails accessibility in dark theme due to incorrect token usage.

### ISSUE THREAD

**Issue by redongreen** [microsoft/fluentui#23879]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-checkbox--default
```

### Reproduction

https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-checkbox--default

### Bug Description

Checkbox fails contrast in dark theme.   Checked-checkbox is currently using `Compound Brand Background (#479ef5)` and `Foreground O Brand (#FFFFFF)`. This combination produces a contrast of 2.8:1. 

For context, this combination worked prior to the brand communication blue update in April due to the saturation value of the ramp.

<img width="296" alt="image" src="https://user-images.githubusercontent.com/20573265/178572916-a3ace5f7-e04b-4b43-92f9-aec2be791763.png">

![image](https://user-images.githubusercontent.com/20573265/178572191-6081e446-9d5b-414d-89cc-711a1858af8d.png)

I wrote a brief guide on where compound brand vs brand background should be applied. [Link to guide](https://www.figma.com/file/xShPc11RUrzGWqZWfR4kFp/Ian-Guisard-Scratchpad?node-id=931%3A198658).

## Actual Behavior
Checked checkbox uses the following background  (s)

Compound Brand Background (rest)
Compound Brand Background (Hover)
Compound Brand Background (Pressed)

## Expected Behavior
Checked checkbox should use the following background(s);

  Brand Background (rest)
Brand Background (Hover)
Brand Background (Pressed)

### Requested priority

High

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by redongreen** [comment]
Closing this issue.

According to Sarah Lee, [Daisy proposed to use inverted colors as foreground](https://www.figma.com/file/edE0nBTGtp5tVToWGNdkNV/Brand-Color?node-id=714%3A258368).
